partner_noncommercial/views/menu.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!--
```

Comment:
You can get rid of these comments as well, as we no longer use the sales_team stuff.